lib/config/migration.ts

```diff
@@ -45,7 +45,7 @@ export function migrateConfig(
       'peerDependencies',
     ];
     const { migratePresets } = GlobalConfig.get();
-    for (const [key, val] of Object.entries(config)) {
+    for (const [key, val] of Object.entries(newConfig)) {
```

Comment:
Are you sure about this change?

Reply:
Yeah, otherwise test fails. Because of
```
...
if (is.object(val)) {
  const subMigrate = migrateConfig(
  migratedConfig[key] as RenovateConfig,
  key
);
...
```
in the end of migration.ts.`val` was took from originalConifg, it is object, but in migrated config this field already removed. Basically no make sense to handle keys, which were removed by MigrationService.

Reply:
That is why failed test coverage, `gitFs` key was already in the list of removed properties in MigraitonService, but  still handles by old script, because I forgot to remove it previously

Reply:
Is this a fix or refactor which should be done independently of the other migrations here?

Reply:
@pret-a-porter Please split those changes our of this PR.

Reply:
Done https://github.com/renovatebot/renovate/pull/14101